Scripts/xbbtools/xbb_blast.py

```diff
@@ -116,9 +116,7 @@ def get_blast_binaries(self):
 
     def database_readable(self, db_paths):
         """Return the name of the blast database without path and extension."""
```

Comment:
Unrelated to this PR, but this should better be "Return list of blast database names without path and extension."